examples/units/basic_units.py

```diff
@@ -345,7 +348,20 @@ def convert(val, unit, axis):
         if units.ConversionInterface.is_numlike(val):
             return val
         if np.iterable(val):
-            return [thisval.convert_to(unit).get_value() for thisval in val]
+            if np.ma.isMaskedArray(val):
```

Comment:
Minor preference for `isinstance(..., MaskedArray)` (on the basis that there's no `isint`, `isfloat`, `isstr` etc functions either...)

Reply:
Done